docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,312 @@
+| `verification_labels` | array | **Required** | The class names from `verification_detector` that count as valid. |
+| `verification_confidence` | float | Optional | The detection confidence needed in order to move into the `DISARMED` state. <br> Default: `0.8` |
+| `countdown_time_s` | int | Optional | The time in seconds the system will remain in state `COUNTDOWN` before transitioning to state `ALARM`. <br> Default: `20` |
+| `alarm_time_s` | int | Optional | The time in seconds the system will remain in  state `ALARM` before transitioning to state `TRIGGER_1`. <br> Default: `10` |
+| `disarmed_time_s` | int | Optional | The time in seconds the system will remain in  state `DISARMED` before transitioning to state `TRIGGER_1`. <br> Default: `10` |
+| `disable_alarm` | bool | Optional | Disables the `COUNTDOWN` and `ALARM` states. The system will always remain in the `TRIGGER_1` and `TRIGGER_2` states. <br> Default: `false` |
+
+## Configure a transform camera
+
+At this point, your machine is fully capable of detecting people in its camera feed, and of identifying whether a specific detected person is "approved" (defined under `"face_labels"`) or not.
+To easily see this in action, you can add a [transform camera](/components/camera/transform/) to your machine to overlay the current state of the on top of the camera feed.
+
+To add a transform camera to your machine:
+
+1. On your machine's **Config** page in the [Viam app](https://app.viam.com), navigate to the **Components** tab.
+1. Click the **Create component** button at the bottom of the page, select `camera`, then select the built-in `transform` model.
+1. Give the transform camera a name, like `my-transform-camera`, then click **Create**.
+1. On the panel that appears, enter the following configuration into the **Attributes** field:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+     "pipeline": [
+       {
+         "attributes": {
+           "classifier_name": "security",
+           "confidence_threshold": 0.5
+         },
+         "type": "classification"
+       }
+     ],
+     "source": "my-webcam"
+   }
+   ```
+
+   If you used different names for the vision service or the camera component, update this configuration with those names.
+   You can adjust the `confidence_threshold` to suit your needs.
+   A value of `0.5` is a relatively loose match, representing 50% confidence.
+
+1. Click **Save Config** at the bottom of the window to save your changes.
+
+## View your verification system in action
+
+{{% alert title="Note" color="note" %}}
+The various states do not cause anything to happen on their own besides appearing as overlays on the transform cam.
+To trigger an audio alarm or otherwise have your machine take an action based on the reported state, you can write your own logic using one of the [Viam SDKs](https://docs.viam.com/build/program/) to [poll the classifications](/ml/vision/#getclassificationsfromcamera).
+{{% /alert %}}
```

Comment:
Hmm, I guess this alert relatively duplicates the first Next Steps. I think it's important enough to be here twice though 👍